extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -101,7 +102,7 @@ interface TransferObject<B> extends SafeCloseable {
          * {@link TransferObject#fetchData(RowSequence)}}. This method should be used when writing unpaginated data, and
          * should not be interleaved with calls to {@link TransferObject#transferOnePageToBuffer()}. Note that this
          * method can lead to out-of-memory error for variable-width types (e.g. strings) if the fetched data is too
-         * big.
+         * long.
```

Comment:
"long" -> "too big to fit in the available heap"?

Reply:
Thank you, this is much better :)